extensions/git/package.nls.json

```diff
@@ -123,6 +123,7 @@
 	"config.ignoreWindowsGit27Warning": "Ignores the warning when Git 2.25 - 2.26 is installed on Windows.",
 	"config.ignoreLimitWarning": "Ignores the warning when there are too many changes in a repository.",
 	"config.ignoreRebaseWarning": "Ignores the warning when it looks like the branch might have been rebased when pulling.",
+	"config.remoteUrlType": "Controls which url type is used to clone a git repository",
```

Comment:
This is the wrong place. This is the `git` extension, but the feature lives in the `github` extension.

Also we should call it with a more meaningful name, like `config.gitProtocol`.

Reply:
That's much better!